content/apps/quotas.md

```diff
@@ -0,0 +1,123 @@
+---
+menu:
+  main:
+    parent: apps
+title: Quotas
+weight: -10
+---
+
+#### Quotas:
+
+Cloud Foundry capacity is billed by quota. Quotas provide a not to exceed reservation of memory, compute, application routes and service instances. Quotas are associated with and billed to a project IAA number or equivalent billing tag. Quota capacity is a hard limit, unused capacity does not ‘roll over’ month to month.
+
+##### Quota Billing:
+
+Quotas are billed by the unit and tallied daily. A monthly bill might include 9 days at 1 unit, 11 days at 2 units and 10 days at 4 units.
+
+##### Quota Units:
+
+Each unit provides.
+
+- 10 application routes.
+- 3.75GB of RAM.
+- 1 vCPU of compute.
```

Comment:
Depends.

The RAM reservation extrapolates to a relative CPU share on the DEA. So there is a guarantee in that sense, but how articulate what means is something I deliberated on for a while. If we stick with M-type instances and the same unit size it remains roughly true.

We could refer to CPU in terms of share more like [Heroku](https://devcenter.heroku.com/articles/dyno-types#available-dyno-types).